lxd/instances_get.go

```diff
@@ -319,7 +323,7 @@ func doInstancesGet(d *Daemon, r *http.Request) (interface{}, error) {
 			}
 
 			for _, inst := range insts {
-				nodeInstances[inst.Name()] = inst
+				nodeInstances[uniqueInstance(inst.Name(), inst.Project())] = inst
```

Comment:
As `nodeInstances` is an internal variable (not returned to client), can we change its definition to `map[[2]string]instance.Instance{}` I think you can do that (can't do it with a slice, but arrays should be ok).
E.g. https://go.dev/play/p/o5hH2W4tceZ

That way we won't have to rely on an arbitrary encoding scheme (which in this case is problematic because project names can contain colons).